supported-configurations.json

```diff
@@ -130,6 +130,22 @@
         "propertyKeys": ["trace_agent_url"]
       }
     ],
+    "DD_TRACE_DEBUG_OPEN_SPANS": [
+      {
+        "version": "A",
+        "type": "boolean",
+        "default_value": "false",
+        "propertyKeys": ["trace_debug_open_spans"]
+      }
+    ],
+    "DD_TRACE_OPEN_SPAN_TIMEOUT": [
```

Comment:
This one is in the wrong alphabetical order. I also think it's weird that it's not named in the same way as `DD_TRACE_DEBUG_OPEN_SPANS`, since it's related. Naming it `DD_TRACE_DEBUG_OPEN_SPANS_TIMEOUT` would make more sense.